src/Jira/Api.php

```diff
@@ -131,54 +133,75 @@ public function getFields()
     }
 
     /**
-     * get specified issue.
-     *
-     * issue key should be YOURPROJ-221
+     * Get specified issue.
      *
-     * @param $issueKey
-     * @param $expand
-     * @return mixed
+     * @param string $issueKey Issue key should be YOURPROJ-221
+     * @param string $expand
+     * @return Result|false
      */
     public function getIssue($issueKey, $expand = '')
     {
         return $this->api(self::REQUEST_GET, sprintf('/rest/api/2/issue/%s', $issueKey), array('expand' => $expand));
     }
 
+    /**
+     * @param string $issueKey
+     * @param array $params
+     *
+     * @return Result|false
+     */
     public function editIssue($issueKey, $params)
     {
         return $this->api(self::REQUEST_PUT, sprintf('/rest/api/2/issue/%s', $issueKey), $params);
     }
 
-
+    /**
+     * @param string $attachmentId
+     *
+     * @return array|false
+     */
     public function getAttachment($attachmentId)
     {
-        $result = $this->api(self::REQUEST_GET, "/rest/api/2/attachment/$attachmentId", array(), true);
-
-        return $result;
+        return $this->api(self::REQUEST_GET, '/rest/api/2/attachment/' . $attachmentId, array(), true);
     }
 
+    /**
+     * @return Result|false
+     */
     public function getProjects()
     {
         return $this->api(self::REQUEST_GET, '/rest/api/2/project');
     }
 
+    /**
+     * @param string $projectKey
+     *
+     * @return array|false
+     */
     public function getProject($projectKey)
     {
-        $result = $this->api(self::REQUEST_GET, "/rest/api/2/project/{$projectKey}", array(), true);
-
-        return $result;
+        return $this->api(self::REQUEST_GET, "/rest/api/2/project/{$projectKey}", array(), true);
```

Comment:
The `$projectKey` variable can be extracted from double-quoted string to allow single quote usage as you did above in `getAttachment` method.